asyncapi.sse.kafka.proxy/k8s/helm/values.yaml

```
@@ -0,0 +1,11 @@
+extraEnv:
+  - name: ZILLA_INCUBATOR_ENABLED
+    value: "true"
```

```suggestion
env:
  ZILLA_INCUBATOR_ENABLED: "true"
```

is incubator needed?